### PR TITLE
MVP Work

### PR DESCRIPTION
**Pull request by mterrel**

### CODE REVIEW

File: packages/df-client/src/views/components/FakeBrowser/fakeBrowser.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
+.browser-wrapper {
```
Comment (Wkasel) [inline packages/df-client/src/views/components/FakeBrowser/fakeBrowser.css:1]: @mterrel is this for the link checker?

File: packages/df-client/src/views/containers/Providers/AppProviders.jsx [hunk 2]
```diff
@@ -10,6 +10,14 @@ import theme, { globalStyles } from '~theme/flux-mui.adapter';
 // import { DrawerProvider } from '~components/Drawer';
 import AppSettingsProvider from './SettingsProvider';
 
+/**
```
Comment (Wkasel) [inline packages/df-client/src/views/containers/Providers/AppProviders.jsx:13]: Good catch, what errors was it causing?

File: packages/df-client/src/views/components/FakeBrowser/fakeBrowser.css (hunk 1 above)
Comment (Wkasel) [reply packages/df-client/src/views/components/FakeBrowser/fakeBrowser.css:1]: Or is it the preview pane?

File: packages/df-client/src/views/containers/Providers/AppProviders.jsx (hunk 2 above)
Comment (mterrel) [reply packages/df-client/src/views/containers/Providers/AppProviders.jsx:13]: It causes a runtime PropTypes error that shows up both on the server for server rendering/static generation and also shows up in the browser console.

File: packages/df-client/src/views/components/FakeBrowser/fakeBrowser.css (hunk 1 above)
Comment (mterrel) [reply packages/df-client/src/views/components/FakeBrowser/fakeBrowser.css:1]: This is for the embedded "web browser" on the restaurant profile sample preview page.

**Pull request merged**